### ISSUE TITLE
Multiple Linear Regression model using Sci-kit Learn

### ISSUE THREAD

**Issue by Abhijit2505** [edualgo/Grokking-Machine-Learning#4]
Add a multiple linear regression model using sci-kit learn library.

## Steps to follow

- Comment about your interest in the issue and the dataset you want to work with
- Tell us your approach in brief
- Let the mentor respond to you and only after that get started with the issue.
- Don't forget to check the already existing model in order to know the structure.

Join the discussion channel here - [eduAlgo Community](https://edualgo-python-community.tribe.so/)
Let's have a chat at the community tab before starting contributions.

**Comment by yashasvimisra2798** [comment]
I can work on this by providing predictions on the *50 Startups* dataset from Kaggle using Multiple Linear Regression. I would like to work on this as a part of DWOC, please assign this issue to me.

**Comment by keshav340** [comment]
I'm interested in this issue and I want to work on the mileage of car prediction using multiple linear regression learn through sci-kit-learn. We download a dataset that is related to a mileage of the car. First of all, we do some graph visualization then, split our data into training and test sets, create a model using the training set, Evaluate your model using the test set, and finally use the model to predict the unknown value. Last but not least we do some performance metrics.

**Comment by Abhijit2505** [comment]
> I can work on this by providing predictions on the _50 Startups_ dataset from Kaggle using Multiple Linear Regression. I would like to work on this as a part of DWOC, please assign this issue to me.

Join the discussion channel here - [eduAlgo Community](https://edualgo-python-community.tribe.so/)
Let's have a chat at the community tab before starting contributions.

**Comment by Abhijit2505** [comment]
> I'm interested in this issue and I want to work on the mileage of car prediction using multiple linear regression learn through sci-kit-learn. We download a dataset that is related to a mileage of the car. First of all, we do some graph visualization then, split our data into training and test sets, create a model using the training set, Evaluate your model using the test set, and finally use the model to predict the unknown value. Last but not least we do some performance metrics.

Join the discussion channel here - [eduAlgo Community](https://edualgo-python-community.tribe.so/)
Let's have a chat at the community tab before starting contributions.

**Comment by Abhijit2505** [comment]
> I can work on this by providing predictions on the _50 Startups_ dataset from Kaggle using Multiple Linear Regression. I would like to work on this as a part of DWOC, please assign this issue to me.

Please get started with the issue

**Comment by Abhijit2505** [comment]
> I'm interested in this issue and I want to work on the mileage of car prediction using multiple linear regression learn through sci-kit-learn. We download a dataset that is related to a mileage of the car. First of all, we do some graph visualization then, split our data into training and test sets, create a model using the training set, Evaluate your model using the test set, and finally use the model to predict the unknown value. Last but not least we do some performance metrics.

Please get started with the issue

**Comment by ksdkamesh99** [comment]
I want to work on it please can you assign me
